packages/node_modules/express-pouchdb/lib/index.js

```diff
@@ -185,6 +185,38 @@ module.exports = function (startPouchDB, opts) {
     next();
   });
 
+  app.use(function (req, res, next) {
+    var regexFindDbName = /\/([^\/]*)/
+    var dbName = decodeURIComponent(req.url.match(regexFindDbName)[1])
+
+    if (!dbName) {
+      return next();
+    }
+
+    // check for internal routes GET /_all_dbs
+    if (dbName[0] === '_') {
+      for (var i = 0, route; i < app._router.stack.length; i++) {
+        route = app._router.stack[i].route;
+        if (route) for (var j = 0, method; j < route.stack.length; j++) {
+          method = route.stack[j].method.toUpperCase()
+          if (method === req.method && route.path.substr(1, dbName.length) === dbName) {
+            console.log('ok?')
+            return next();
+          }
+        };
+      }
+    }
```

Comment:
I can’t patch PouchDB so that it renames the db name, because I use `express-pouchdb` within another app, and I use the same constructor inside this app. When I do `new PouchDB('user/abc456')` I must get the database that `express-pouchdb` created on `PUT /user%2Fabc456` – or am I missing something?